### ISSUE TITLE
Facebook pages only?

### ISSUE THREAD

**Issue by LiamWinder** [anantgarg/socialturn#5]
Hi, got this up and running. I notice when I connect to Facebook I have to choose a users page to manage, I can't post directly to the users page. Is there any way around this?

**Comment by anantgarg** [comment]
Can you please elaborate?

**Comment by LiamWinder** [comment]
Yep no problem. So say I add my Facebook account in "Connect" I then have to choose a Facebook page to manage, so a page that I have created within Facebook. But on Facebook most people do not create pages, they simply post to their own timeline.

In Facebook you can create multiple pages and manage them all yourself. The issue here is that it is only presenting an option to post to one of these user created pages, not the actual timeline itself.

Hope that makes sense?

**Comment by anantgarg** [comment]
Currently ST supports FB pages only. Will add support for FB user timelines in the near future.

**Comment by LiamWinder** [comment]
Great. When I get more time I'll take a look too. Its a great app, very impressed!
